tests/System.IO.Pipelines.Performance.Tests/columns/RpsColumn.cs

```diff
@@ -21,10 +21,22 @@ public string GetValue(Summary summary, Benchmark benchmark)
 
         public bool IsDefault(Summary summary, Benchmark benchmark) => false;
         public bool IsAvailable(Summary summary) => true;
+
+        public string GetValue(Summary summary, Benchmark benchmark, ISummaryStyle style)
```

Comment:
Since you have to change this, lets get rid of it altogether and replace it with this https://github.com/aspnet/KestrelHttpServer/blob/c2f15fcac374328d8256b8345bfb5eb22e8add75/test/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/CoreConfig.cs#L19